FireRender.Maya.Src/Context/TahoeContext.cpp

```diff
@@ -241,6 +241,10 @@ void TahoeContext::setupContext(const FireRenderGlobalsData& fireRenderGlobalsDa
 	frstatus = rprContextSetParameterByKey1u(frcontext, RPR_CONTEXT_METAL_PERFORMANCE_SHADER, fireRenderGlobalsData.useMPS ? 1 : 0);
 	checkStatus(frstatus);
 
+	MStringArray textureCachePath;
```

Comment:
Why optionVar command is still used? Its not necessary. You should use just attrbiute